### ISSUE TITLE
Tooltip does not stay open on mouse hover

### ISSUE THREAD

**Issue by mimarz** [digdir/designsystemet#2941]
User should be able to move hover from the anchor to the tooltip, and it should stay open.
This is known missing functionality from #2916

Ref:
https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html
https://www.uutilsynet.no/veiledning/1413-pekerfolsomt-innhold-eller-innhold-ved-tastaturfokus/1243#:~:text=Mulig%20%C3%A5%20bruke%20med%20mouseover%3A%E2%80%AFDersom%20mouseover%20f%C3%A5r%20ekstra%20innhold%20til%20%C3%A5%20komme%20til%20syne%2C%20kan%20pekeren%20flyttes%20over%20det%20ekstra%20innholdet%20uten%20at%20innholdet%20forsvinner
